lib/http/node.js

```diff
@@ -55,9 +55,10 @@ AWS.NodeHttpClient = AWS.util.inherit({
 
     // connection timeout support
     if (httpOptions.connectTimeout) {
+      var connectTimeoutId;
```

Comment:
Based on the usage of `connectTimeoutId` in the stream `error` handler below, do you think it would be clearer if this declaration were moved outside of the `if (httpOptions.connectTimeout)` block? I don't think it will have a functional difference given `var` is function scoped - just clarity.